check.go

```diff
@@ -131,10 +129,6 @@ func NewCheckCreateInputTypeOf[T any](checkCreateInput CheckCreateInput) *T {
 	return newCheck
 }
 
-func (c *CheckCreateInput) GetCheckCreateInput() *CheckCreateInput {
```

Comment:
Removed this along with `GetCheckUpdateInput` because there are no use cases for both of these functions.

Reply:
This is used in the terraform provider and is the basis for `CheckCreateInputProvider` up above